Packs/Carbon_Black_Enterprise_Response/IncidentFields/incidentfield-IOC_Type.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
when creating incident fields within a pack. please prefix the pack with the integration/pack name to ensure uniqueness

Reply:
will change to incidentfield-CarbonBlackEDR_IOC_Type.json